hawkular-alerts-rest/hawkular-alerts-rest-api/src/main/java/org/hawkular/alerts/rest/AlertsHandler.java

```diff
@@ -171,6 +178,8 @@ public Response findAlerts(
             final UriInfo uri) {
         Pager pager = RequestUtil.extractPaging(uri);
         try {
+            ResponseUtil.checkForUnknownQueryParams(uri, PARAMS_FIND_ALERTS);
```

Comment:
So, basically the approach is to maintain a list of parameters and check manually internally.
I had the hope that RESTEasy could offer some help for this, but if not it sounds fine.